ambry-messageformat/src/test/java/com.github.ambry.messageformat/BlobStoreHardDeleteTest.java

```diff
@@ -84,8 +85,7 @@ public BlobStoreHardDeleteTest(short headerVersionToUse) {
               blobVersions[0], blobTypes[0]);
 
       MessageFormatInputStream msg1 =
-          getPutMessage(keys[1], ByteBuffer.wrap(encryptionKey), blobProperties, usermetadata, blob, BLOB_SIZE,
-              blobVersions[1], blobTypes[1]);
+          getPutMessage(keys[1], null, blobProperties, usermetadata, blob, BLOB_SIZE, blobVersions[1], blobTypes[1]);
```

Comment:
minor: can you add a comment that this msg tests null encryptionKye and next one tests with encryptionKey